doc/reference/tt_cli/configuration.rst

```diff
@@ -0,0 +1,98 @@
+tt CLI configuration
+====================
+
+.. _tt-config_file:
+
+Configuration file
+------------------
+
+The ``tt`` configuration file defines aspects of its runtime, such as directories
+that it uses and log retention parameters.
+
+By default, the configuration file is called ``tarantool.yaml``. The location
+where ``tt`` searches it depends on the :ref:`launch mode <tt-config_modes>`.
+You can also pass the configuration file explicitly in the ``--cfg``
+:doc:`argument <arguments>`.
+
+``tt`` configuration file is a YAML file with the following content:
+
+..  code:: yaml
+
+    tt:
+        modules:
+            directory: path/to/modules/dir
+        app:
+            instances_available: path/to/available/applications
+            run_dir: path/to/run_dir
+            log_dir: path/to/log_dir
+            log_maxsize: num (MB)
+            log_maxage: num (Days)
+            log_maxbackups: num
+            restart_on_failure: bool
+
+modules section
+~~~~~~~~~~~~~~~
+
+* ``directory`` -- a directory where external modules are stored.
+
+app section
+~~~~~~~~~~~
+
+*   ``instances_available`` -- a directory where :ref:`instances <admin-instance_file>`
+    are stored.
+*   ``run_dir``-- a directory for instance runtime artifacts, such as console
+    sockets or PID files.
+*   ``log_dir`` -- a directory where log files are stored.
```

Comment:
@LeonidVas could you help with the default values?

Reply:
By default the current directory will be used, but it will be changed soon.